tests/foreman/api/test_contentmanagement.py

```diff
@@ -72,6 +73,7 @@ class ContentManagementTestCase(APITestCase):
     """
 
     @tier2
+    @pytest.mark.skip("Uses old large_errata repo from repos.fedorapeople")
```

Comment:
Hello @Griffin-Sullivan, first a minor point: your initial comment only mentions SRPM and DRPM repos. You should mention why you are making this change. More important, as this test is for the "ContentManagement" component we should ensure @latran knows about this change (He is primary owner of "Content Management" tests).

Reply:
@swadeley @Griffin-Sullivan has this been resolved?

Reply:
@JacobCallahan I updated the commit message, and I'm waiting for @latran to look everything over. I'm working on changing this line altogether, as we will use the https://fixtures.pulpproject.org/rpm-long-updateinfo/ repo to replace large_errata.

Reply:
@Griffin-Sullivan - content management is disabled/discontinued since February.  I plan on either deleting the entirety of test_contentmanagement.py and moving the test cases to the specific component area where it suits the test case.  It is best to put your test case in ```test_errata.py```.